travis_configure.sh

```diff
@@ -15,27 +15,30 @@ print_error_msg ()
 {
     test ! -z "$1" || exit 1
     echo "$1"
-    echo "  os:      $TRAVIS_OS_NAME"
-    echo "  package: $DIST_PACKAGE_TARGET"
+    echo "  os:         $TRAVIS_OS_NAME"
+    echo "  lsb:        $DIST_NAME $DIST_VERSION"
+    echo "  targets:    $MAKE_TARGET"
 
     return 0
 }
 
 # Script which is run before the installation script is called
 #
-# Installs dependencies
-# linux:
-#   deb:
-#     - fpm
-# osx:
+# For linux based builds docker-compose is used to set up the build environment
+#
+# For osx based builds homebrew is used to install required packages:
 #   - sed, (libtool), openssl, mysql
 # Returns 0
 before_install()
 {
     if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
-        if [[ "$DIST_PACKAGE_TARGET" == "DEB" ]]; then
-            gem install fpm -v 1.8.1
+        # build images
+        docker-compose -f ./docker-compose.yml \
+            -f ./docker/docker-compose.$DIST_NAME.$DIST_VERSION.yml \
+            build
 
+        # jfrog dependency
+        if [[ "$MAKE_TARGET" =~ "deb" ]]; then
             if [[ -n "$TRAVIS_TAG" && $TRAVIS_REPO_SLUG == "sociomantic-tsunami/libdrizzle-redux" ]]; then
                 curl -XGET -L -k 'https://api.bintray.com/content/jfrog/jfrog-cli-go/$latest/jfrog-cli-linux-amd64/jfrog?bt_package=jfrog-cli-linux-amd64' > /tmp/jfrog ;
```

Comment:
Maybe you want to use `sociomantictsunami/base` image now (it comes with `jfrog`)? Or was there a problem with that?

Reply:
The `base` image doesn't have any D stuff :)

Reply:
Ah, got things confused a bit after looking at cachalot 😀